index.bs

```diff
@@ -2453,6 +2453,8 @@ Unanimous Short Circuit</h5>
 	the Team recommends a resolution
 	and every potential member of a Council who is not renouncing their seat
 	votes affirmatively (no abstentions) to adopt this resolution.
+	As an exception, Tim Berners-Lee <em class=rfc2119>may</em> abstain
+	without blocking the adoption of the resolution.
```

Comment:
If we take #791 as introducing the concept of a standing _invitation_, then this could be a better way to address the general problem of invited members of TAG being on the blocking path:

```suggestion
	and every potential member of a Council who is not renouncing their seat
	or declining their invitation
	votes affirmatively (no abstentions) to adopt this resolution.
```

Reply:
This PR and #791 are alternatives to eachother. We should take or or the other (or #792), but there's not need to take both.